realm/realm-jni/src/io_realm_internal_Util.cpp

```diff
@@ -78,7 +80,9 @@ JNIEXPORT jlong JNICALL Java_io_realm_internal_Util_nativeGetMemUsage(JNIEnv*, j
 JNIEXPORT jstring JNICALL Java_io_realm_internal_Util_nativeGetTablePrefix(
     JNIEnv* env, jclass)
 {
-    return to_jstring(env, string(TABLE_PREFIX));
+    string table_prefix(TABLE_PREFIX);
```

Comment:
Removing the `std` name space will probably required that we write `std::size_t` many places.